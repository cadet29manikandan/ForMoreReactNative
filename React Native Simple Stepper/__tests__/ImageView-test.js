import React from 'react';
import ImageView from '../src/ImageView';
import renderer from 'react-test-renderer';

describe('ImageView', () => {
  describe('renders', () => {
    test('snapshot', () => {
      const wrapper = renderer.create(<ImageView />).toJSON();
      expect(wrapper).toMatchSnapshot();
    });
  });
});
