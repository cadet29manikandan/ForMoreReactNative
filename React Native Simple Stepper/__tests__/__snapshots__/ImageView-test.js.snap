// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ImageView renders snapshot 1`] = `
<Image
  source={Object {}}
  style={
    Array [
      Object {
        "height": 36,
        "width": 36,
      },
      Object {
        "opacity": 1,
      },
    ]
  }
/>
`;
